### PR TITLE
[39850] Apply two column class initially only when the view is wide enough

### PR DESCRIPTION
**Pull request by HDinger**
https://community.openproject.org/projects/openproject/work_packages/39850/activity

### CODE REVIEW

File: frontend/src/app/features/work-packages/components/wp-single-view/wp-single-view.component.ts [hunk 1]
```diff
@@ -267,6 +267,10 @@ export class WorkPackageSingleViewComponent extends UntilDestroyedMixin implemen
     return this.I18n.t('js.project.work_package_belongs_to', { projectname: project });
   }
 
+  showTwoColumnLayout():boolean {
+    return this.$element[0].getBoundingClientRect().width > 750;
```
Comment (Kharonus) [inline frontend/src/app/features/work-packages/components/wp-single-view/wp-single-view.component.ts:271]: seems like a magic number to me. is `750` used elsewhere? could we reuse it?

**Review by machisuji — APPROVED** [review]
👍

File: frontend/src/app/features/work-packages/components/wp-single-view/wp-single-view.component.ts (hunk 1 above)
Comment (Kharonus) [reply frontend/src/app/features/work-packages/components/wp-single-view/wp-single-view.component.ts:271]: but this is minor

**Review by Kharonus — APPROVED** [review]
Tested. Bug behavior is fixed

**Pull request merged**